### ISSUE TITLE
salt-minion service is not created for Amazon Linux AMI 2 when running bootstrap-salt.sh git master

### ISSUE THREAD

**Issue by janniswarnat** [saltstack/salt-bootstrap#1895]
### Description of Issue/Question

I am trying to install a salt-minion on an Amazon EC2 instance (Amazon Linux AMI 2) and the bootstrap script fails to create the salt-minion service. The script clones the Salt git repository and looks for the service definition file at `/tmp/git/salt/pkg/rpm/salt-minion.service` but the correct place would be `/tmp/git/salt/pkg/common/salt-minion.service`. Let me know if I can create a pull request for simply replacing `rpm` with `common`. 

### Steps to Reproduce Issue
```
git clone https://github.com/saltstack/salt-bootstrap.git
cd salt-bootstrap
sudo sh ./bootstrap-salt.sh git master
 *  INFO: Running version: 2022.10.04
 *  INFO: Executed by: sh
 *  INFO: Command line: './bootstrap-salt.sh -D git master'
 *  WARN: Running the unstable version of bootstrap-salt.sh

 *  INFO: System Information:
 *  INFO:   CPU:          GenuineIntel
 *  INFO:   CPU Arch:     x86_64
 *  INFO:   OS Name:      Linux
 *  INFO:   OS Version:   5.10.157-139.675.amzn2.x86_64
 *  INFO:   Distribution: Amazon Linux AMI 2

 * DEBUG: Binaries will be searched using the following $PATH: /sbin:/bin:/usr/sbin:/usr/bin
 *  INFO: Installing minion

[...]

Successfully installed salt-3006.0+0na.be5707d
 *  INFO: Checking if Salt can be imported using python3

Installed Salt Version: 3006.0+0na.be5707d
Installed Salt Package Path: /usr/lib/python3.7/site-packages/salt

 *  INFO: Running install_amazon_linux_ami_2_git_post()
 *  WARN: /tmp/git/salt/pkg/rpm/salt-minion.service does not exist!
 *  INFO: Running install_amazon_linux_ami_2_check_services()
 * DEBUG: Checking if service salt-minion is enabled
Failed to get unit file state for salt-minion.service: No such file or directory
 * DEBUG: Service salt-minion is NOT enabled
 * ERROR: Failed to run install_amazon_linux_ami_2_check_services()!!!
 * DEBUG: Cleaning up the Salt Temporary Git Repository
 * DEBUG: Removing the logging pipe /tmp/bootstrap-salt.logpipe
 * DEBUG: Removing the temporary apt error file /tmp/apt_error.bmzevk
```

### Versions and Systems
See above

**Comment by janniswarnat** [comment]
File `salt-minion.service` was removed from folder `pkg/rpm` in this [commit](https://github.com/saltstack/salt/commit/701633d1814e0be0dd388ae22ad11a71c11beb9f).

**Comment by janniswarnat** [comment]
Fixed via 4eabf0aeb6b5bea2884cbfab64af4ec00acc8b90.
